Source/DataSources/NodeTransformation.js

```diff
@@ -0,0 +1,61 @@
+/*global define*/
+define([
+        '../Core/Cartesian3',
+        '../Core/defaultValue',
+        '../Core/Matrix4',
+        '../Core/Quaternion'
+    ], function(
+        Cartesian3,
+        defaultValue,
+        Matrix4,
+        Quaternion) {
+    "use strict";
+
+    var defaultScale = new Cartesian3(1.0, 1.0, 1.0);
+    var defaultTranslation = Cartesian3.ZERO;
+    var defaultRotation = Quaternion.IDENTITY;
+
+    /**
+     * A set of transformations to apply to a particular node in a {@link Model}.
+     * @alias NodeTransformation
+     * @constructor
+     *
+     * @param {Cartesian3} [scale=new Cartesian3(1.0, 1.0, 1.0)] A {@link Cartesian3} specifying the (x, y, z) scaling to apply to the node.
+     * @param {Cartesian3} [translation=Cartesian3.ZERO] A {@link Cartesian3} specifying the (x, y, z) translation to apply to the node.
+     * @param {Quaternion.IDENTITY} [rotation=Quaternion.IDENTITY] A {@link Quaternion} specifying the (x, y, z, w) rotation to apply to the node.
+     */
+    var NodeTransformation = function(scale, translation, rotation) {
```

Comment:
I understand why this is in DataSources, but it certainly looks like something I would except to be in Core.  Are we sure that we're never going to need something like this at the lower level?